addons/website/static/src/scss/website.scss

```diff
@@ -1018,11 +1019,29 @@ header {
                     padding-right: .5em;
                 }
             }
-
+            @if o-website-value('header-template') == 'vertical' {
+                .oe_structure_solo {
+                    // Hide the separator. Otherwise it inverts the alignment.
+                    display: none;
+                }
+            }
```

Comment:
Ok I must be missing something ^^ In standard 14.0, the alignment has no effect on my side... neither does he with your branch :grimacing: 

In any case, this is a violet change in stable: what if the user added a product searchbar in that oe_structure_solo? We cannot make it suddenly disappear :thinking:

Reply:
Without this "display: none", the product search will be outside the "off-canvas" menu ([screenshot](https://imgur.com/a/IWELFm7)).
Indeed, it might be better to move the "oe_structure" inside the "off-canvas" menu (on opening) as we have already done in some headers But in the case of the separator (default template => therefore used by most users), it has no interest and it's ugly.

Reply:
We cannot suppose the separator was kept by most users and thus allow ourselves to break other users (even if that were true, we cannot break 1% of the user while there is a possibility to not break anybody with something "less beautiful") IMO. Also if that was really the concern, why not just hide the separator when it is alone in that oe_structure instead of the whole oe_structure? :thinking: 

But if I understand you right, that oe_structure section is broken for all off-canvas menu whether or not it is a separator or something else, right? If yes, then it's ok to hide it when in off-canvas of course but I don't understand why this is not the explanation above the rule then? I don't understand what "Otherwise it inverts the alignment" mean actually :grimacing:

Reply:
Actually, the "oe_structure solo" being the 1st element inside the "top_menu_collapse" which has a "display: flex" position. And not being inside the menu, it pushes this one in 2nd position with the "left" alignment. Or in first position, when the "rows" are reversed with the "right" alignment. The solution is therefore to hide it because putting it in the menu would not make sense, especially when it is a separator. I updated the comments.